### ISSUE TITLE
Use the Pointer Events API

### ISSUE THREAD

**Issue by bates64** [towerofnix/macbook-pressure-art#1]
For some reason this project uses the Safari Force Touch API or something... _why?_ The **standardized** API for detected pointer events, is, of course, the Pointer Events API... and [it supports pressure](https://developer.mozilla.org/en-US/docs/Web/API/PointerEvent/pressure). It works in all browsers, including IE 10... except Safari doesn't support it yet, obviously. <sup>Thanks, Apple!</sup>

**Comment by towerofnix** [comment]
Quoting sidebar:

![Sidebar note](https://cloud.githubusercontent.com/assets/9948030/23321950/1ea9b94c-fab8-11e6-8b29-fddfcaa34e70.png)

But thanks for the issue, I've added it to a milestone so I can remember to do this! :snail:

**Comment by bates64** [comment]
Couldn't you, like, fallback to the Safari API and use the Pointer Event `.pressure` property when its available _now_? 📦

**Comment by towerofnix** [comment]
No, because I can't test it. The only browser that tracks MacBook pressure input is Safari, and it only sends it through its own API.

Say what you will, or "USE PRESSURE.JS :angry:", but I'm not going to send off any untested code to the internets!

OTOH, I'll update the milestone. Really I should do this when any browser supports the normal pressure API with MacBooks, okay? Because then I can test it.
